plugins/bundle/config_test.go

```diff
@@ -24,12 +27,25 @@ func TestConfigValidation(t *testing.T) {
 			input:   `{"name": a/b/c", "service": "service2"}`,
 			wantErr: false,
 		},
+		{
+			input:   `{"name": a/b/c", "service": "service2", "prefix: mybundle"}`,
```

Comment:
```suggestion
			input:   `{"name": "a/b/c", "service": "service2", "prefix": "mybundle"}`,
```

Looks like this copied an issue in the test above. Could you correct that one and the one below as well?

Reply:
sure, added more tests to catch this scenario.
Done